rxjs-playground/src/app/exercises/dragdrop/dragdrop.component.ts

```diff
@@ -35,7 +37,9 @@ export class DragdropComponent {
 
     /******************************/
 
-    
+    this.mouseDown$.subscribe(_ => this.subscription = this.mouseMove$.subscribe(mouseMove => this.setTargetPosition(mouseMove)));
```

Comment:
Kreative Lösung 😇 Es gibt noch andere Wege, um Subscriptions zu beenden, als unsubscribe() zu benutzen, z.B. einen passenden Operator, den du direkt in der Pipeline einsetzen kannst.
Subscribe in Subscribe ist eher kein so guter Stil.
Du kannst ja mal versuchen, die Lösung nochmal anders zu bauen!